vital/types/detected_object.cxx

```diff
@@ -68,10 +85,27 @@ ::clone() const
   new_obj->m_index = this->m_index;
   new_obj->m_detector_name = this->m_detector_name;
   new_obj->m_descriptor = this->m_descriptor;
+  new_obj->m_geo_point = this->m_geo_point;
```

Comment:
I *think* this is okay, but please consider: if I clone a `detection`, someone that has the original detection must not be able to modify the clone's geo-point. (Again, I *think* it is okay, I just want to be sure...)

Reply:
The geo_point is not a pointer, so it's copying the value to a new object, so the two values are independent of each other